Basic-Car-Maintenance/Shared/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -26,6 +26,7 @@ final class SettingsViewModel {
         "bugReport": URL(string: "https://github.com/mikaelacaron/Basic-Car-Maintenance/issues")!
     ]
     
+    /// Fetches the list of contributors for the GitHub repository 'Basic-Car-Maintenance'.
```

Comment:
```suggestion
    /// Fetches the list of contributors for the GitHub repository [Basic-Car-Maintenance](https://github.com/mikaelacaron/Basic-Car-Maintenance).
```

This adds a hyperlink to the repo in the DocC documentation